pkg/coord/adapter.go

```diff
@@ -32,7 +32,22 @@ func (a *adapter) ShareKeyRange(id string) error {
 	return fmt.Errorf("shareKeyRange not implemented")
 }
 
-func (a *adapter) ListKeyRanges(ctx context.Context) ([]*kr.KeyRange, error) {
+func (a *adapter) ListKeyRanges(ctx context.Context, _ string) ([]*kr.KeyRange, error) {
```

Comment:
Why not accept dataspace parameter here?

Reply:
Fixed